include/kernel-version.mk

```diff
@@ -4,11 +4,11 @@ LINUX_RELEASE?=1
 
 LINUX_VERSION-3.18 = .43
 LINUX_VERSION-4.4 = .53
-LINUX_VERSION-4.9 = .14
+LINUX_VERSION-4.9 = .16
 
 LINUX_KERNEL_HASH-3.18.43 = 1236e8123a6ce537d5029232560966feed054ae31776fe8481dd7d18cdd5492c
 LINUX_KERNEL_HASH-4.4.53 = 7efecdd5d47c173fc54bbefa5f78e31782532cfa8dbc0df47473e87e35cdf3e6
-LINUX_KERNEL_HASH-4.9.14 = 534d343f4ca5c55d7214694a2a378897e8ee2adf576563f069da010093b3b640
+LINUX_KERNEL_HASH-4.9.16 = 43b444915cbde67d47b6b614e277c14856a55a63698054298be3a16fe3babae4
```

Comment:
i'm so stupid.... changed the kernel-version in the compile dir but not here ahahahah